Update the guidance on Default Azure Credential usage (#267)
microsoft/agent-skills: .github/plugins/azure-sdk-typescript/skills/azure-storage-file-share-ts/SKILL.md

);
```

> ### Entra ID TokenCredential (Recommended)

Reviewer: ```suggestion
### Microsoft Entra Token Credential (Recommended)
```

## Best Practices

> 1. **Use Entra ID TokenCredential for production** — Enable managed identity in Azure

Reviewer: ```suggestion
1. **Use Microsoft Entra Token Credential for production** — Enable managed identity in Azure
```

| `downloadToFile()` | ✅ | ❌ |
| `downloadToBuffer()` | ✅ | ❌ |
| SAS generation | ✅ | ❌ |

> | Entra ID TokenCredential | ✅ | ❌ |

Reviewer: ```suggestion
| Microsoft Entra Token Credential | ✅ | ❌ |
```
